lib/rucio/core/config.py

```diff
@@ -435,6 +466,31 @@ def read_from_cache(key: str, expiration_time: int = 900) -> Any:
     return value
 
 
+@overload
+def write_to_cache(key: Literal['sections'], value: list[str]) -> None:
+    ...
+
+
+@overload
+def write_to_cache(key: OptionsKey, value: list[str]) -> None:
+    ...
```

Comment:
Can be unified into a single overload:

```suggestion
@overload
def write_to_cache(key: Union[OptionsKey, Literal['sections']], value: list[str]) -> None:
    ...
```